packages/stellar_solana_bridge/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:18 AS build
+
+WORKDIR /app
+
+COPY . .
+# Install yarn and build project
+RUN yarn install
+RUN yarn lerna run build --no-private && yarn workspace @threefold/tft_solana build
```

Comment:
this will just build the package and if we run a container from this image it will stop immediately. It should have an entrypoint or CMDs. please check the other packages and how they work. (for example: https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/playground/Dockerfile)